app/controllers/dashboards_controller.rb

```diff
@@ -23,6 +21,7 @@ def build
     end
 
     redirect_to dashboards_path
+    `rake fix_coord_mismatch`
```

Comment:
It'd probably be best that this be ran in a background job as this will block web workers.

Reply:
Or perhaps a cron job?

Reply:
do we have a worker set up?